client/src/components/util/truncate.js

```diff
@@ -85,13 +85,30 @@ export default (props) => {
       "aria-label": originalString,
     })
   );
+
+  // we need an ID to check for this content, since this is the only place the geneset description appears
+  const descriptionContent = (
+    <span
+      test-id={`geneset-description-tooltip-${originalString}${tooltipAddendum}`}
+    >
+      {originalString}
+      {tooltipAddendum}
+    </span>
+  );
+
   return (
     <Tooltip2
-      content={`${originalString}${tooltipAddendum}`}
+      content={
+        isGenesetDescription
+          ? descriptionContent
+          : `${originalString}${tooltipAddendum}`
+      }
```

Comment:
Why not always wrap it in a span and optionally pass a test-id prop to keep it more agnostic

Reply:
Snapshots.

Reply:
But it would still show in the snapshot, can you write an issue to not couple the truncate component to geneset descriptions.